src/Microsoft.Health.Dicom.Core/Features/CustomTag/CustomTagStoreEntry.cs

```diff
@@ -10,11 +10,12 @@ namespace Microsoft.Health.Dicom.Core.Features.CustomTag
     /// </summary>
     public class CustomTagStoreEntry
     {
-        public CustomTagStoreEntry(int key, string path, string vr, CustomTagLevel level, CustomTagStatus status)
+        public CustomTagStoreEntry(int key, string path, string vr, string privateCreator, CustomTagLevel level, CustomTagStatus status)
```

Comment:
Should there be more validation of the input?

Reply:
CustomTagStoreEntry just stand for a single row in database.